v4/mobile/react-native/FastLink/package.json

```diff
@@ -2,9 +2,13 @@
 	"name": "FastLink",
 	"version": "0.0.1",
 	"private": true,
+	"engines": {
+		"node": "16.x"
```

Comment:
Fixes this error spat out by Metro:

<img width="591" alt="Screenshot 2024-07-15 at 15 46 23" src="https://github.com/user-attachments/assets/a80b6a61-2113-4eef-9c67-f9030ea274ad">